### PR TITLE
Update "Dropping old Python" with `universal=0`

### LINKED ISSUE #726: advice for maintainers on dropping universal=1 wheels
There should be some advice on what to do about universal wheels when dropping support for python versions in this guide: https://packaging.python.org/guides/dropping-older-python-versions/

eg, should this:
```
# setup.cfg
[bdist_wheel]
universal=1
```
 be removed?

### PR DESCRIPTION
**Pull request by webknjaz**
Resolves #726

### CODE REVIEW

**Comment by webknjaz** [comment]
@gaborbernat any feedback? :point_up:

**Comment by webknjaz** [comment]
cc @graingert @hugovk

File: source/guides/dropping-older-python-versions.rst [hunk 1]
```diff
@@ -22,6 +22,33 @@ This workflow requires that:
 2. The latest version of :ref:`twine` is used to upload the package,
 3. The user installing the package has at least Pip 9.0, or a client that supports the Metadata 1.2 specification.
 
+Dealing with the universal wheels
+---------------------------------
+
+Traditionally, projects providing Python code that is semantically
+compatible with both Python 2 and Python 3, produce :term:`wheels
+<Wheel>` that have ``py2.py3`` tag in their names. When dropping
```
Comment (hugovk) [inline source/guides/dropping-older-python-versions.rst:30]:
```suggestion
<Wheel>` that have a ``py2.py3`` tag in their names. When dropping
```

File: source/guides/dropping-older-python-versions.rst [hunk 2]
```diff
@@ -22,6 +22,33 @@ This workflow requires that:
...
+support for Python 2, it is important not to forget to change this tag
+to just ``py3``. It is often configured within :file:`setup.cfg` under
+the ``[bdist_wheel]`` section by setting ``universal = 1`` :ref:`if they
+use setuptools <Universal Wheels>`.
+
+If you use this method, either remove the said option or section, or
+set the ``universal`` setting to ``0`` explicitly:
```
Comment (hugovk) [inline source/guides/dropping-older-python-versions.rst:37]:
```suggestion
If you use this method, either remove this option or section, or
explicitly set ``universal`` to ``0``:
```

**Review by hugovk — APPROVED** [review]
Looks good, just a few minor wording suggestions!

File: source/guides/dropping-older-python-versions.rst (hunk 2 above)
Comment (webknjaz) [reply source/guides/dropping-older-python-versions.rst:37]: By the way, maybe we should also mention `tag = py3`?

**Review by gaborbernat — APPROVED** [review]

**Review by pradyunsg — APPROVED** [review]

**Pull request merged**